src/bin/stratisd.rs

```diff
@@ -35,6 +38,29 @@ fn print_err(err: StratisError) -> () {
     eprintln!("{}", err.description());
 }
 
+/// Given a udev event check to see if it's an add and if it is return the device node and
+/// device number.
+fn handle_udev_add(event: &libudev::Event) -> Option<(PathBuf, u64)> {
+    match event.event_type() {
```

Comment:
Instead of a full match, since you are only interested in Add event, should do:

```
if event.event_type() == libudev::EventType::Add {
    // do stuff
}
```